src/openrct2/ride/RideData.h

```diff
@@ -77,6 +77,12 @@ enum class RideColourKey : uint8_t
     Toilets
 };
 
+enum class TrackDesignCreateMode : uint64_t
```

Comment:
Any reason for `uint64_t`? Unless it is to avoid replays breaking and stuff, it seems a bit of an overkill, could have been `bool`

Reply:
This looks like a nice opportunity to use `int_fast8_t`.